### PR TITLE
Fixes #13842: Use openssl 1.0.2 on old agents

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/13842

### CODE REVIEW

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -28,8 +28,12 @@ FUSION_RELEASE = 2.3.19
 FUSION_SHA1 = a01af094613a69feac778c2402cb639cb227eb1f
 LMDB_RELEASE = 0.9.22
 LMDB_SHA1 = 5d62d4c7527f3474f60a0d87db2bfd539e6493eb
+# Default openssl version to use
 OPENSSL_RELEASE = 1.1.1
 OPENSSL_SHA1 = e4559f31dca37ce815e0c7135488b747745a056d
+# open ssl version to use on older systems
+OLD_OPENSSL_RELEASE = 1.0.2p
```
[the remark this answers is not in the document]
Comment (amousset) [reply rudder-agent/SOURCES/Makefile]: I don't know if it worth it

**Comment by peckpeck** [comment]
Missing the debian part

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
I told about the debian part on Slack ;) And it's now done

File: rudder-agent/SOURCES/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (VinceMacBuche) [reply rudder-agent/SOURCES/Makefile]: I don't think, we want the most general case to be the default case even if we have merge issue

**Comment by peckpeck** [comment]
Ok, I thought there was some problem on very old debian too

**Pull request merged**